crates/astria-conductor/src/tendermint.rs

```diff
@@ -30,12 +36,8 @@ impl ValidatorSet {
     /// the validator with the highest proposer priority is the proposer.
     /// TODO: could there ever be two validators with the same priority?
     pub(crate) fn get_proposer(&mut self) -> eyre::Result<Validator> {
-        self.validators.sort_by(|v1, v2| {
-            v1.proposer_priority
-                .parse::<i64>()
-                .unwrap()
-                .cmp(&v2.proposer_priority.parse::<i64>().unwrap())
-        });
+        self.validators
+            .sort_by(|v1, v2| v1.proposer_priority.0.cmp(&v2.proposer_priority.0));
```

Comment:
Not relevant for this PR but this `get_proposers` function is doing too much/should not have side effects. It should probably either just sort or use `iter().max_by` to find the proposer. Or at least be renamed.

But yeah, not relevant for this PR/